resources/troubleshooter/data.json

```diff
@@ -433,7 +429,7 @@
                 },
                 {
                     "answer": "The camera takes a long time to take a photo.",
-                    "message": "Try reducing the resolution in your code, you'll have to find an optimum resolution for your robot to get the best speed and detail."
+                    "message": "The vision system always takes longer for the first photo. Consider taking a photo when your robot first starts."
```

Comment:
Is resolution no longer configurable? If it is configurable, then that might still be something we want to mention here.

Reply:
It is no longer configurable, but performance is sufficiently high that it's not such an issue.